pebbles/static/partials/blueprints.html

```diff
@@ -55,6 +63,8 @@ <h3>Blueprints</h3>
                     <button ng-click="openReconfigureBlueprintDialog(blueprint)" class="btn btn-default">Reconfigure</button>
                     <button ng-show="blueprint.is_enabled" ng-click="openBlueprintLinkDialog(blueprint)" class="btn btn-success">Link</button>
                     <button ng-confirm-click="Copy this blueprint?" ng-click="copyBlueprint(blueprint)" class="btn btn-default">Copy</button>
+                    <button ng-confirm-click="Do you want to permanently archive this blueprint?" ng-click="archiveBlueprint(blueprint)" class="btn btn-default">Archive</button>
```

Comment:
Archive: this feature is only for the admins, the group owners should not see this option

Reply:
Ok.